### DISCUSSION TITLE
Discussion platform

### DISCUSSION THREAD

**Discussion by JacquesCarette** [JacquesCarette/Drasil#3236] [Polls]
I'd like to be able to send messages to everyone that works on Drasil, in a more ephemeral way sometimes. For this, I've used Discord with my other teams. It would also be possible to use Slack or Teams. Or even Mattermost. So I'm going to poll.

**Comment by balacij** [comment]
I like this idea!

Truthfully, I would prefer we avoided Teams. It has a lot of problems (e.g., crashing, minor visual bugs and artifacts, slow, resource heavy, tied to McMaster's authentication policy, uncertainty with adding external users, we wouldn't be able to have a public instance that the Drasil community could use/communicate with us, etc.).

I think Discord is the most convenient option.

**Comment by smiths** [comment]
I don't have much experience with Discord or Slack, but I agree with @balacij about avoiding Teams.  I'm not a fan of Teams.  Although I don't use it, I have a positive impression of Discord from what I hear people say, so that is where I cast my vote.  :smile:

**Comment by JacquesCarette** [comment]
The vote is clear, Discord it is. I'll set up an instance.

**Comment by cd155** [comment]
Sorry for the late response. I would vote for Slack though the vote is obvious. The reason I chose Slack is that it is very friendly for software developers. They have a lot of integration with GitHub. Code snip and code review are also quite friendly. They also have some pre-program bots to check the code stylishly.
